README.md

```diff
@@ -101,6 +101,9 @@ redirect: async () => {
 **ATTENTION**: The factory function **must** return an array with redirect
 objects (as seen above).
 
+## Gotcha's
+The redirect module only works in universal mode (will not work on generate). Redirects are realized through a server middleware, which can only react when there is a server running which is only the case when your application is in universal/SSR mode.
```

Comment:
```suggestion
The redirect module will not work in combination with `nuxt generate`. Redirects are realized through a server middleware, which can only react when there is a server running.
```

You can have an SPA running with a server behind it (like you have it with the default SSR setup) :relaxed: